src/tpm2.c

```diff
@@ -108,11 +114,15 @@ static int TPM2_CommandProcess(TPM2_CTX* ctx, TPM2_Packet* packet,
     CmdInfo_t* info, TPM_CC cmdCode, UINT32 cmdSz)
 {
     int rc = TPM_RC_SUCCESS;
-    UINT32 authSz, handleValue;
+    UINT32 authSz;
     BYTE *param, *encParam = NULL;
     int paramSz, encParamSz = 0;
-    int i, authPos, handlePos;
+    int i, authPos;
     int tmpSz = 0; /* Used to calculate the new total size of the Auth Area */
+#ifndef WOLFTPM2_NO_WOLFCRYPT
```

Comment:
I'm seeing a bunch of errors with:
`./configure CFLAGS="-DWOLFTPM2_NO_WOLFCRYPT" && make`
Is that the intended usage?

```
  CC       src/src_libwolftpm_la-tpm2_param_enc.lo
In file included from ./wolftpm/tpm2_param_enc.h:26:0,
                 from src/tpm2_param_enc.c:22:
./wolftpm/tpm2_packet.h:57:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 WOLFSSL_LOCAL void TPM2_Packet_InitBuf(TPM2_Packet* packet, byte* buf, int size);
               ^~~~
In file included from src/tpm2_packet.c:23:0:
./wolftpm/tpm2_packet.h:57:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 WOLFSSL_LOCAL void TPM2_Packet_InitBuf(TPM2_Packet* packet, byte* buf, int size);
               ^~~~
src/tpm2_packet.c:85:6: error: no previous prototype for ‘TPM2_Packet_InitBuf’ [-Werror=missing-prototypes]
 void TPM2_Packet_InitBuf(TPM2_Packet* packet, byte* buf, int size)
      ^~~~~~~~~~~~~~~~~~~
Makefile:1994: recipe for target 'src/src_libwolftpm_la-tpm2_param_enc.lo' failed
make[1]: *** [src/src_libwolftpm_la-tpm2_param_enc.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
cc1: all warnings being treated as errors
In file included from src/tpm2.c:24:0:
./wolftpm/tpm2_packet.h:57:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 WOLFSSL_LOCAL void TPM2_Packet_InitBuf(TPM2_Packet* packet, byte* buf, int size);
               ^~~~
Makefile:1973: recipe for target 'src/src_libwolftpm_la-tpm2_packet.lo' failed
make[1]: *** [src/src_libwolftpm_la-tpm2_packet.lo] Error 1
In file included from ./wolftpm/tpm2_param_enc.h:26:0,
                 from src/tpm2_wrap.c:23:
./wolftpm/tpm2_packet.h:57:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 WOLFSSL_LOCAL void TPM2_Packet_InitBuf(TPM2_Packet* packet, byte* buf, int size);
               ^~~~
In file included from ./wolftpm/tpm2_tis.h:26:0,
                 from src/tpm2_tis.c:23:
./wolftpm/tpm2_packet.h:57:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 WOLFSSL_LOCAL void TPM2_Packet_InitBuf(TPM2_Packet* packet, byte* buf, int size);
               ^~~~
Makefile:1980: recipe for target 'src/src_libwolftpm_la-tpm2_tis.lo' failed
make[1]: *** [src/src_libwolftpm_la-tpm2_tis.lo] Error 1
src/tpm2.c: In function ‘TPM2_GetNonce’:
src/tpm2.c:5383:13: error: implicit declaration of function ‘TPM2_Packet_InitBuf’ [-Werror=implicit-function-declaration]
             TPM2_Packet_InitBuf(&packet, buffer, (int)sizeof(buffer));
             ^~~~~~~~~~~~~~~~~~~
src/tpm2.c:5383:13: error: nested extern declaration of ‘TPM2_Packet_InitBuf’ [-Werror=nested-externs]
Makefile:1987: recipe for target 'src/src_libwolftpm_la-tpm2_wrap.lo' failed
make[1]: *** [src/src_libwolftpm_la-tpm2_wrap.lo] Error 1
cc1: all warnings being treated as errors
Makefile:1966: recipe for target 'src/src_libwolftpm_la-tpm2.lo' failed
make[1]: *** [src/src_libwolftpm_la-tpm2.lo] Error 1
make[1]: Leaving directory '/home/pi/git/wolfTPM'
Makefile:1397: recipe for target 'all' failed
make: *** [all] Error 2
```